devtools/conda-envs/basic.yaml

```diff
@@ -28,18 +27,13 @@ dependencies:
 
   # openmmforcefields brings in the full toolkit; if that is ever re-packaged
   # this should be changed to openff-toolkit-base
-  - openff-toolkit >= 0.11.3
-  # argcomplete (brought in via BSE) pulls down totally incorrect builds of `importlib_metadata`,
-  - importlib_metadata >=4
-  - importlib-metadata >=4
-  - rdkit
```

Comment:
This is brought in by the toolkit's recipe (since the base recipe is not used). Arguably it's worth including here to be explicit, I guess